WikiSentiment/WikiAPIRequests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json.Nodes;
+using System.Text.Json;
+
+namespace WikiSentiment
+{
+    /// <summary>
+    /// Collection of API requests that parse json data
+    /// </summary>
+    public static class WikiAPIRequests
+    {
+        /// <summary>
+        /// Gets english title of given article. Returns empty string if none exist
+        /// </summary>
+        /// <param name="_client">http client, api headers required</param>
+        /// <param name="_title"></param>
+        /// <param name="_countryCode">two letter language code</param>
+        /// <returns>English title, returns empty if none</returns>
+        public static async Task<string> GetLangLink(
+                HttpClient _client, string _title, string _countryCode)
+        {
+            string languageTarget = "en";
+            _title = stripSubmenuLink(_title);
+            JsonNode langlinkNode;
+
+            string url = $"https://{_countryCode}.wikipedia.org/w/api.php?action=query&titles=" +
+                $"{_title}&prop=langlinks&format=json&lllang={languageTarget}";
+
+            var langlinkResponse = await _client.GetStringAsync(url);
+            var jsonObject = JsonNode.Parse(langlinkResponse);
+
+            //validate json response, return empty string if failed
+            if (!jsonObject.AsObject().ContainsKey("query") ||
+                !JsonNode.Parse(langlinkResponse).AsObject()["query"].AsObject().ContainsKey("pages"))
```

Comment:
```suggestion
            if (!(jsonObject.AsObject().ContainsKey("query") &&
                  JsonNode.Parse(langlinkResponse).AsObject()["query"].AsObject().ContainsKey("pages")))
```
Reasonable people might disagree.